lib/adapters/code-action-adapter.ts

```diff
@@ -27,7 +27,7 @@ export default class CodeActionAdapter {
   public static async getCodeActions(
     connection: LanguageClientConnection,
     serverCapabilities: ServerCapabilities,
-    linterAdapter: LinterPushV2Adapter | null,
+    linterAdapter: LinterPushV2Adapter | undefined ,
```

Comment:
`ActiveServer.linterPushV2` is typed as `LinterPushV2Adapter | undefined`, that gets passed into this method for this parameter